packages/psss/src/containers/NavBar.js

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
This is not a new component, it has just been moved from `components` to `containers` and connected to redux